cmd/agentctl/commands/log.go

```diff
@@ -61,69 +59,52 @@ A CLI tool to connect to vppagent and show vppagent logs.
 }
 
 type LogListOptions struct {
-	Logger string
+	Name string
 }
 
-func RunLogList(cli *AgentCli, opts LogListOptions) error {
-	resp, err := cli.GET("/log/list")
-	if err != nil {
-		return fmt.Errorf("HTTP GET request failed: %v", err)
-	}
-	Debugf("%s", resp)
+func RunLogList(cli agentcli.Cli, opts LogListOptions) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	msg := string(resp)
-	if strings.Contains(msg, "404 page not found") {
-		fmt.Println(msg)
-		return fmt.Errorf("not found")
-	}
-
-	data, err := utils.ConvertToLogList(msg)
+	loggers, err := cli.Client().LoggerList(ctx, types.LoggerListOptions{
+		Name: opts.Name,
```

Comment:
second argument will be ignored by `Client().LoggerList()`, so maybe no need to initialize struct with some values?